public/app/percona/shared/components/PageSwitcherCard/PageSwitcherCard.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = ({ v1: { spacing } }: GrafanaTheme2) => ({
+  pageSwitcherWrapper: css`
+    display: flex;
+    padding: ${spacing.lg} 0px ${spacing.lg} 0px;
+    margin-bottom: ${spacing.lg};
+    gap: 10px;
+  `,
+  wrapper: css`
+    width: 380px;
```

Comment:
What if we will have the big length of strings on cards?

Reply:
It will stretch vertically. I used the same style as used in grafana.